radix-engine/src/kernel/call_frame.rs

```diff
@@ -454,6 +456,7 @@ impl CallFrame {
         }
     }
 
+    #[trace_resources(info = "CallFrame")]
```

Comment:
Let's trace every `pub`/`pub (crate)` method in `CallFrame`.

Reply:
Removed tracing from call_frame.rs as this is not needed I think, also this produced lots of data.